tests/foreman/cli/test_errata.py

```diff
@@ -1275,3 +1280,112 @@ def test_positive_check_errata_dates(module_org):
     # Verify any errata UPDATED date from stdout
     validate_updated_date = datetime.strptime(result[0]['updated'], '%Y-%m-%d').date()
     assert isinstance(validate_updated_date, date)
+
+
+@pytest.fixture(scope='module')
+def rh_repo_module_manifest(module_manifest_org):
+    """Use module manifest org, creates RH tools repo, syncs and returns RH repo."""
+    # enable rhel repo and return its ID
+    rh_repo_id = enable_rhrepo_and_fetchid(
+        basearch=DEFAULT_ARCHITECTURE,
+        org_id=module_manifest_org.id,
+        product=PRDS['rhel'],
+        repo=REPOS['rhst7']['name'],
+        reposet=REPOSET['rhst7'],
+        releasever=None,
+    )
+    # Sync step because repo is not synced by default
+    rh_repo = entities.Repository(id=rh_repo_id).read()
+    rh_repo.sync()
+    return rh_repo
+
+
+"""Section for tests using RHEL7.7 Content Host.
+   The applicability tests using Default Content View are related to the introduction of Pulp3.
+   """
+
+
+@pytest.fixture(scope='module')
+def default_contentview(module_manifest_org):
+    return entities.ContentView(organization=module_manifest_org, name=DEFAULT_CV).search()
+
+
+@pytest.fixture(scope='module')
+def new_module_ak(module_manifest_org, rh_repo_module_manifest, default_lce):
+    new_module_ak = entities.ActivationKey(
+        content_view=module_manifest_org.default_content_view,
+        environment=entities.LifecycleEnvironment(id=module_manifest_org.library.id),
+        organization=module_manifest_org,
+    ).create()
+    # Ensure tools repo is enabled in the activation key
+    new_module_ak.content_override(
+        data={'content_overrides': [{'content_label': REPOS['rhst7']['id'], 'value': '1'}]}
+    )
+    # Fetch available subscriptions
+    subs = entities.Subscription(organization=module_manifest_org).search()
+    assert len(subs) > 0
+    # Add default subscription to activation key
+    sub_found = False
+    for sub in subs:
+        if sub.name == DEFAULT_SUBSCRIPTION_NAME:
+            new_module_ak.add_subscriptions(data={'subscription_id': sub.id})
+            sub_found = True
```

Comment:
Non blocking, but this block seems a bit clunky, starting with getting the subscriptions.

The only subscription you need here is the default subscription, and then this block exists to find that, and then add it to the ak.

Consider a pattern more like the following, which is much more succinct and I believe achieves the same things as L1325-1334

```
subs = entities.Subscription(organization=module_manifest_org).search(
    query={'search': f'name={DEFAULT_SUBSCRIPTION_NAME}'}
)
assert subs
new_module_ak.add_subscriptions(data={'subscription_id': subs[0].id})

return new_module_ak

```

Reply:
Hello, that code works but only when used without `name=`, for some reason search is only working with `name=` if the search string is double quoted. I think its safe to leave it out.

Reply:
ack, yeah full disclose that was typed without running it - was focusing more on the pattern at hand then the specific behavior of search.  Sounds a bit odd though, but if you can reliably filter that instance in the search call, I would.

Reply:
OK, I have made that change, thank you